specs/switch.md

```diff
@@ -77,10 +81,10 @@ Typically it's best to include a checked and unchecked message but it is not req
 ```
 <fast-switch
     checked={true}
-    unchecked-message="On"
-    checked-message="Off"
 >
     Notify by Email
+    <span class="content" slot="checked-message">On</span>
+    <span class="content" slot="unchecked-message">Off</span>
```

Comment:
Can you remove the classes as part of the change?

Reply:
I like your idea, thanks for the suggestion.